site/lib/_sass/components/_tooltip.scss

```
@@ -0,0 +1,57 @@
+.tooltip-wrapper {
+    position: relative;
+
+    a.tooltip-target {
+        color: inherit;
+        text-decoration: underline dotted;
+    }
+
+    .tooltip {
+        visibility: hidden;
+
+        display: flex;
+        position: absolute;
+        z-index: 100;
+        top: 100%;
+        left: 50%;
+        transform: translateX(-50%);
+
+        flex-flow: column nowrap;
+        width: 250px;
+
+        background: var(--site-raised-bgColor);
+        border: 0.05rem solid rgba(0, 0, 0, .125);
+        border-radius: 0.75rem;
+        box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, .15);
+        padding: 0.8rem;
+
+        .tooltip-header {
+            font-size: 1.2rem;
+            font-weight: 500;
+            margin-bottom: 0.25rem;
+        }
+
+        .tooltip-content {
+            font-size: 0.875rem;
+            color: var(--site-secondary-textColor);
+        }
+    }
+
+    // On non-touch devices, show tooltip on hover or focus.
+    @media all and not (pointer: coarse) {
+        &:hover .tooltip {
+            visibility: visible;
+        }
+
+        &:has(.tooltip-target:focus) .tooltip {
```

Can this just use [`:focus-within`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within) that way it is also visible if focus is within the tooltip?

```suggestion
        &:focus-within .tooltip {
```

---

Done